extensions/vertx-http/runtime/src/main/java/io/quarkus/vertx/http/runtime/VertxHttpRecorder.java

```diff
@@ -67,6 +67,8 @@
 
     private static volatile Handler<RoutingContext> hotReplacementHandler;
 
+    private static volatile String websocketSubProtocols;
```

Comment:
This should not be a field, it should just be passed in as a parameter to the method that requires it.

Reply:
the reason I end up adding an attribute is because of the restart on fail. If I just send this to the start server part, It wouldnt be able to have the proper state when trying to recover for the restart.

Reply:
Hmm, either way this is not going to work 100% reliably in the restart on failure case, as if that other method gets called will depend on when it fails.

Reply:
@stuartwdouglas ok, I removed the variable.